src/nvim/api/command.c

```diff
@@ -634,6 +634,21 @@ String nvim_cmd(uint64_t channel_id, Dict(cmd) *cmd, Dict(cmd_opts) *opts, Arena
   build_cmdline_str(&cmdline, &ea, &cmdinfo, args);
   ea.cmdlinep = &cmdline;
 
+  // Check for "++opt=val" argument.
+  if (ea.argt & EX_ARGOPT) {
+    while (ea.arg[0] == '+' && ea.arg[1] == '+') {
+      if (getargopt(&ea) == FAIL && !is_cmd_ni(ea.cmdidx)) {
+        api_set_error(err, kErrorTypeValidation, "Invalid argument");
```

Comment:
can we use `api_err_invalid` or `VALIDATE_S` so that the invalid arg is mentioned in he message?